addons/music/fnc_getMusicPath.sqf

```diff
@@ -0,0 +1,28 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicPath
```

Comment:
Again header line-breaks missing. I won't comment that anymore for all the other files this applies to